fy_pm_controller:
  type: position_controllers/JointTrajectoryController
  joints:
     - joint1
     - joint2
     - joint3
     - joint4
     - joint5   #this write style is same as joints: ['joint_1','joint_2','joint_3','joint_4']

    #  - joint6
    #  - joint7
  constraints:
      goal_time: 0.6
      stopped_velocity_tolerance: 0.01

  stop_trajectory_duration: 0.0
  state_publish_rate:  50
  action_monitor_rate: 20

gains:
        joint1:   {p: 3.0, i: 0.01, d: 1.1, i_clamp: 1.0}
        joint2:   {p: 3.0, i: 0.01, d: 1.1, i_clamp: 1.0}
        joint3:   {p: 3.0, i: 0.01, d: 1.1, i_clamp: 1.0}
        joint4:   {p: 3.0, i: 0.01, d: 1.1, i_clamp: 1.0}
        joint5:   {p: 3.0, i: 0.01, d: 1.1, i_clamp: 1.0}
        # joint6:   {p: 3.0, i: 0.01, d: 1.0, i_clamp: 1.0}
        # joint7:   {p: 3.0, i: 0.01, d: 1.0, i_clamp: 1.0}

# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
